readme.md

```diff
@@ -12,6 +12,16 @@ This repository provides a mechanism for community members to share their Views
 
 The Analytics Middle Tier is an open source project and these views are also available as open source. By default all contributions are available under the terms of the [Apache License, version 2.0](license). Please ensure that you maintain the appropriate licensing to contribute this work for others to reuse. If you wish to use terms other than the Apache 2.0 license then please include the license file with your views.
 
+## Installation
```

Comment:
We need to include the following information:

- Any pre-requisites that may be needed to run the installer
- Any configuration files that need to be updated: which files, which properties, and why document which scripts to run, the order of the scripts that need to be run, any query parameters that need to be entered